### PR TITLE
[#158110992] Insert SPID certificates generation step in README.md

### PR DESCRIPTION
**Pull request by lmorelli986**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🐞 [#158110992](https://www.pivotaltracker.com/story/show/158110992): Negli Installation steps del README manca la generazione dei certificati per SPID locale

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: README.md [hunk 1]
```diff
@@ -97,18 +97,19 @@ A Linux/macOS environment is required at the moment.
...
-```
-localhost    spid-testenv-identityserver
-localhost    italia-backend
-```
-
-7. wait a couple of minutes to let the IDP start (or monitor the process with `$ tail -f logs/idp/wso2carbon.log`)
-8. run `scripts/import-spid-data.sh` to configure the local IDP
-9. copy `app/.env.example` to `app/.env` and fill the variables with your values
-10. point your browser to [https://italia-backend](https://italia-backend)
+8. run `yarn generate:test-certs` to create SAML (SPID) certificates
```
Comment (gunzip) [inline README.md]: FYI, if use the markdown notation 

```markdown
1.   first list item
1.   second list item
1.   third list item
```

you don't need to reorder the numbers everytime you change the list :-)

Comment (gunzip) [inline README.md]: we should have a docker task even for this step othwerwise when working on win32 you run the other commands inside the docker linux container, then the procedure will fail here running yarn from the win32 host.

Comment (lmorelli986) [reply README.md]: You are right. I did it just for matching the pre-compiled version to the expected result.

**Pull request merged**